Testing (#31)
YunoHost-Apps/movim_ynh: doc/DISCLAIMER.md

## Configuration

>  * How to configure this app: From an admin panel at this address https://example.com/movim/?admin. Username and password are defined during installation.

Reviewer: ```suggestion
 * After installing the app, it can be configured at https://example.com/movim/?admin using the username and password defined during the installation.
```
